features/customer.feature

```diff
@@ -91,3 +91,26 @@ Scenario: Search for john
 #     Then I should see the message "Success"
 #     And I should see "Loki" in the results
 #     And I should not see "fido" in the results
+
+Scenario: Deleting a customer
+    When I visit the "home page"
+    And I set the "first_name" to "sally"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    And I should see "sally" in the "first_name" field
+    When I press the "Delete" button
+    Then I should see the message "Success"
+    And the "customer_Id" field should be empty
```

Comment:
Two issues here:
1. the full field name is "customer_id", but we pre-pend the "customer_" id prefix to these fields (see `web_steps.py` line 34), so this should just be `"id"`
2. currently, the customer_id field is not being emptied. you will have to update the `clear_form_data` function in `rest_api.js` to clear the "customer_id" field